jbmc/unit/java_bytecode/goto_program_generics/mutually_recursive_generics.cpp

```diff
@@ -10,11 +10,14 @@ Author: Diffblue Ltd.
 #include <java-testing-utils/require_goto_statements.h>
 #include <java-testing-utils/require_type.h>
 #include <testing-utils/use_catch.h>
+#include <util/config.h>
 
 SCENARIO(
   "Generics class with mutually recursive_generic parameters",
   "[core][java_bytecode][goto_programs_generics]")
 {
+  config.ansi_c.set_LP64();
+
```

Comment:
:pick: Why are these needed? Especially, why is the one here required?

Reply:
Without it there is failure when simplifying 0+0, because certain indexes have width 0 and the simplify_plus method checks the widths are non-zero. I assume it is needed now because some things became pointers that weren't before.